From b62503c578604bef65a7e00da4f48a7876bc260c Mon Sep 17 00:00:00 2001
From: Lars Op den Kamp <lars@opdenkamp.eu>
Date: Thu, 13 Sep 2012 14:00:54 +0200
Subject: [PATCH] [rpi/cec] and now correct, use CEC_RPI_VIRTUAL_COM from
 cectypes.h (which also uses the correct value instead of
 'CEC')

---
 xbmc/peripherals/bus/linux/PeripheralBusRPi.cpp |    4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/xbmc/peripherals/bus/linux/PeripheralBusRPi.cpp b/xbmc/peripherals/bus/linux/PeripheralBusRPi.cpp
index 08e4190..8a436e4 100644
--- a/xbmc/peripherals/bus/linux/PeripheralBusRPi.cpp
+++ b/xbmc/peripherals/bus/linux/PeripheralBusRPi.cpp
@@ -19,6 +19,7 @@
  */
 
 #include "PeripheralBusRPi.h"
+#include <libcec/cectypes.h>
 
 extern "C" {
 #include <interface/vmcs_host/vc_cecservice.h>
@@ -29,7 +30,6 @@
 
 #define RPI_PERIPHERAL_BUS_VID 0x2708
 #define RPI_PERIPHERAL_CEC_PID 0x1001
-#define RPI_PERIPHERAL_CEC_LOC "CEC"
 
 CPeripheralBusRPi::CPeripheralBusRPi(CPeripherals *manager) :
     CPeripheralBus(manager, PERIPHERAL_BUS_RPI)
@@ -45,7 +45,7 @@ bool CPeripheralBusRPi::PerformDeviceScan(PeripheralScanResults &results)
     result.m_iVendorId   = RPI_PERIPHERAL_BUS_VID;
     result.m_iProductId  = RPI_PERIPHERAL_CEC_PID;
     result.m_type        = PERIPHERAL_CEC;
-    result.m_strLocation = RPI_PERIPHERAL_CEC_LOC;
+    result.m_strLocation = CEC_RPI_VIRTUAL_COM;
 
     if (!results.ContainsResult(result))
       results.m_results.push_back(result);
-- 
1.7.10

